int32 robot_id
string status 

